dojo/fixtures/sla_configurations.json

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "model": "dojo.sla_configuration",
+        "pk": 1,
+        "fields": {
+            "name": "Default",
+            "description": "The Default SLA Configuration. Products not using an explicit SLA Configuration will use this one.",
+            "critical": 7,
+            "enforce_critical": true,
+            "high": 30,
+            "enforce_high": true,
+            "medium": 90,
+            "enforce_medium": true,
+            "low": 120,
+            "enforce_low": true,
+            "async_updating": false
+        }
+    },
+    {
+        "model": "dojo.sla_configuration",
+        "pk": 100,
```

Comment:
I think you can leave this out and Django will automatically assign the next available ID when the fixture is loaded.

Hopefully `100` is safe if we decide to keep it... ;)